scripts/fetch_and_build_rpm

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 
 # make bash behave
-set -euo pipefail
+set -eo pipefail
+# In one branch we execute commands inside One Branch steps, since One Branch does not allow executing docker inside
+# docker. Additionally, Onebranch needs containers not to close so we make it hang for OneBranch to be able to
+# execute commands.
+if [[ -z ${CONTAINER_BUILD_RUN_ENABLED} ]]; then
+  echo "INFO: Image working in waiting mode. Not executing build script"
+  tail -f /dev/null
+  exit 0
```

Comment:
If this is an error, does it make sense to return 0?

Reply:
Also, is this even reachable? How do we terminate or stop this docker image?

Reply:
You're right I added it just to show that it is not error and a valid code path. It's better to remove it. It may be confusing

Reply:
How do we terminate the docker container then? Does it keep running forever indefinitely?

Reply:
docker stop :)